builtin/providers/aws/resource_aws_dynamodb_table.go

```diff
@@ -158,6 +159,20 @@ func resourceAwsDynamoDbTable() *schema.Resource {
 					return hashcode.String(buf.String())
 				},
 			},
+			"stream_enabled": &schema.Schema{
+				Type: schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"stream_view_type": &schema.Schema{
+				Type: schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
There are only 4 values so I can certainly add a small func that will check it's the correct values. It wouldn't take me long if you feel it would be useful